pkg/ovs/openflow/utils.go

```diff
@@ -1226,6 +1227,23 @@ func FlowModMatchString(flowMod *openflow15.FlowMod) string {
 	return fmt.Sprintf("table=%d,%s", flowMod.TableId, getFlowModMatch(flowMod))
 }
 
+func FlowDumpMatchString(flowMod *openflow15.FlowMod) string {
```

Comment:
I personally find the naming a bit confusing. Additionally, would you consider using an additional parameter for `FlowModMatchString` instead?
Either something like this:
```go
func FlowModMatchString(flowMod *openflow15.FlowMod, omitPriority bool) string {
}
```
or 
```go
func FlowModMatchString(flowMod *openflow15.FlowMod, omitFields ...string) string {
}
```

Reply:
Sure I'll see how to best refactor this